### PR TITLE
PUD-122 upload email

### PR DESCRIPTION
**Pull request by jonnywyatt**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -100,6 +100,7 @@ jobs:
     executor:
       name: hmpps/node
       tag: 14.16-browsers
+    resource_class: xlarge
```
Comment (philsegal) [inline .circleci/config.yml:103]: 😮

File: fake-manage-recalls-api/stubs/__files/get-recall.json [hunk 2]
```diff
@@ -12,6 +12,11 @@
     {
       "category": "PART_A_RECALL_REPORT",
       "documentId": "34bdf-5717-4562-b3fc-2c963f66afa6"
+    },
+    {
+      "category": "RECALL_NOTIFICATION_EMAIL",
+      "documentId": "64bdf-3455-8542-c3ac-8c963f66afa6",
+      "fileName": "2021-07-03 Phil Jones recall.msg"
```
Comment (philsegal) [inline fake-manage-recalls-api/stubs/__files/get-recall.json:19]: pedantic, but aren't we only supporting .eml?

File: server/views/pages/assessEmail.njk [hunk 3]
```diff
@@ -65,6 +63,25 @@
...
+                            label: {
+                                text: "Upload the email"
+                            },
+                            hint: {
+                                html: govukInsetText({
+                                    text: "Save the email to your computer then upload the file."
+                                })
+                            },
+                            attributes: {
+                                accept: '.msg'
```
Comment (philsegal) [inline server/views/pages/assessEmail.njk:78]: do we have a test to show that you cant upload a non '.msg' file?

**Review by philsegal — APPROVED** [review]

File: server/views/pages/assessEmail.njk (hunk 3 above)
Comment (jonnywyatt) [reply server/views/pages/assessEmail.njk:78]: no, I'll add one

**Pull request merged**